### ISSUE TITLE
Avg NaN - classifier training on Windows 8.1

### ISSUE THREAD

**Issue by mickes27** [AlexeyAB/darknet#1182]
Hello. I compiled darknet on Windows 8.1 and everything was going well until I tried to train my own classifier. I just want to classify object, it's position is not important so I am not training YOLO, but I followed this one:

https://pjreddie.com/darknet/train-cifar/

The files I created (all with Linux-style new lines):

data/cifar/labels.txt

```
airplane
automobile
bird
cat
deer
dog
frog
horse
ship
truck


```

data/cifar/test.list

```
...
D:/Artificial Intelligence/Gotowe sieci/YOLO v3/build/darknet/x64/data/cifar/test/9787_ship.png
D:/Artificial Intelligence/Gotowe sieci/YOLO v3/build/darknet/x64/data/cifar/test/6261_airplane.png
D:/Artificial Intelligence/Gotowe sieci/YOLO v3/build/darknet/x64/data/cifar/test/5582_deer.png
D:/Artificial Intelligence/Gotowe sieci/YOLO v3/build/darknet/x64/data/cifar/test/827_frog.png
...
```
data/cifar/train.list

```
...
D:/Artificial Intelligence/Gotowe sieci/YOLO v3/build/darknet/x64/data/cifar/train/25658_dog.png
D:/Artificial Intelligence/Gotowe sieci/YOLO v3/build/darknet/x64/data/cifar/train/22929_cat.png
D:/Artificial Intelligence/Gotowe sieci/YOLO v3/build/darknet/x64/data/cifar/train/17229_truck.png
D:/Artificial Intelligence/Gotowe sieci/YOLO v3/build/darknet/x64/data/cifar/train/48406_frog.png
...
```
cfg/cifar.data

```
classes=10
train  = data/cifar/train.list
valid  = data/cifar/test.list
labels = data/cifar/labels.txt
backup = backup/
top=2


```

cfg/cifar_small.cfg

```
[net]
batch=128
subdivisions=1
height=28
width=28
channels=3
max_crop=32
min_crop=32

hue=.1
saturation=.75
exposure=.75

learning_rate=0.1
policy=poly
power=4
max_batches = 5000
momentum=0.9
decay=0.0005

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=16
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=1
stride=1
pad=1
activation=leaky

[convolutional]
filters=10
size=1
stride=1
pad=1
activation=leaky

[avgpool]

[softmax]

```

But when I am using

darknet.exe classifier train cfg/cifar.data cfg/cifar_small.cfg

I am getting:

```
Loaded: 0.000000 seconds
81, 0.207: -nan(ind), -nan(ind) avg, 0.093676 rate, 0.009000 seconds, 10368 imag
es
```

everytime -nan(ind) avg. What did I do wrong (on VirtualBox Ubuntu training works well). Is this encoding error? Maybe my filepaths or new line chars are wrong?

**Comment by Jped** [comment]
Hi I am having the same problem as you here #1185 . How did you label the images? Did you make a txt file for each image?

**Comment by mickes27** [comment]
I followed directly mentioned tutorial. I had only one file named labels.txt
